polkadot/node/core/pvf/src/artifacts.rs

```diff
@@ -75,6 +76,8 @@ pub struct ArtifactId {
 
 impl ArtifactId {
 	const PREFIX: &'static str = "wasmtime_";
+	// FIXME or just "v" instead?
+	const NODE_VERSION_PREFIX: &'static str = "polkadot_v";
```

Comment:
I'd keep like this. Maybe in the future we can also add the `wasmtime` version. So it will look like `wasmtime_vXXX_polkadot_vYYY_...`